```console
Building CXX object CMakeFiles/cmTC_00946.dir/ossp_uuid_lib_test.cc.o


  /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/g++-13
  -isystem /usr/include/uuid -o
  CMakeFiles/cmTC_00946.dir/ossp_uuid_lib_test.cc.o -c
  /tmp/drogon-20230824-6746-jd94ug/cmake/tests/ossp_uuid_lib_test.cc

  /tmp/drogon-20230824-6746-jd94ug/cmake/tests/ossp_uuid_lib_test.cc: In
  function 'int main()':

  /tmp/drogon-20230824-6746-jd94ug/cmake/tests/ossp_uuid_lib_test.cc:5:5:
  error: 'uuid_create' was not declared in this scope; did you mean
  'uuid_clear'?

      5 |     uuid_create(&uuid);
        |     ^~~~~~~~~~~
        |     uuid_clear

  /tmp/drogon-20230824-6746-jd94ug/cmake/tests/ossp_uuid_lib_test.cc:6:21:
  error: 'UUID_MAKE_V1' was not declared in this scope

      6 |     uuid_make(uuid, UUID_MAKE_V1);
        |                     ^~~~~~~~~~~~

  /tmp/drogon-20230824-6746-jd94ug/cmake/tests/ossp_uuid_lib_test.cc:6:5:
  error: 'uuid_make' was not declared in this scope; did you mean
  'uuid_type'?

      6 |     uuid_make(uuid, UUID_MAKE_V1);
        |     ^~~~~~~~~
        |     uuid_type

  make[1]: *** [CMakeFiles/cmTC_00946.dir/build.make:78:
  CMakeFiles/cmTC_00946.dir/ossp_uuid_lib_test.cc.o] Error 1

  make[1]: Leaving directory
  '/tmp/drogon-20230824-6746-jd94ug/build/cmaketest/CMakeFiles/CMakeTmp'

  make: *** [Makefile:127: cmTC_00946/fast] Error 2
```

--- a/.github/workflows/homebrew.yml
+++ b/.github/workflows/homebrew.yml
@@ -15,15 +15,33 @@ jobs:
           - macos-11
           - macos-12
           - macos-13
-          # FIXME: -- Found UUID: /home/linuxbrew/.linuxbrew/lib/libuuid.so
-          # CMake Error at CMakeLists.txt:218 (message):
-          #   uuid lib error
-          # - ubuntu-18.04
-          # - ubuntu-20.04
-          # - ubuntu-22.04
+          - ubuntu-20.04
+          - ubuntu-22.04
     steps:
     - uses: actions/checkout@v3
-    - run: brew update
-    - run: brew install --build-from-source --debug --verbose ./Formula/drogon.rb
-    - run: brew test Formula/drogon.rb
-    - run: brew audit --verbose --strict drogon
+
+    - name: brew update
+      run: |
+        [ -d "/home/linuxbrew/.linuxbrew" ] && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+        brew update
+
+    - name: brew install
+      run: |
+        [ -d "/home/linuxbrew/.linuxbrew" ] && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+        brew install --build-from-source --debug --verbose ./Formula/drogon.rb
+
+    - name: brew link binutils
+      if: matrix.os == 'ubuntu-20.04'
+      run: |
+        [ -d "/home/linuxbrew/.linuxbrew" ] && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+        brew link --force binutils
+
+    - name: brew test
+      run: |
+        [ -d "/home/linuxbrew/.linuxbrew" ] && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+        brew test --verbose Formula/drogon.rb
+
+    - name: brew audit
+      run: |
+        [ -d "/home/linuxbrew/.linuxbrew" ] && eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+        brew audit --verbose --strict drogon

--- a/Formula/drogon.rb
+++ b/Formula/drogon.rb
@@ -3,8 +3,8 @@ class Drogon < Formula
   homepage "https://drogon.org"
   # pull from git tag to get submodules
   url "https://github.com/drogonframework/drogon.git",
-      tag:      "v1.8.6",
-      revision: "f361995035bac47cbae76fb12d9f2f0fa3ef28dc"
+      tag:      "v1.8.7.rc1",
+      revision: "366311c19686eb31601d19d74618b8250da33c88"
   license "MIT"
   head "https://github.com/drogonframework/drogon.git"
 
@@ -16,7 +16,13 @@ class Drogon < Formula
   depends_on "zlib"
 
   def install
-    system "cmake", "-B", "build", *std_cmake_args
+    cmake_args = std_cmake_args
+    if OS.linux?
+      cmake_args << "-DUUID_LIBRARIES=uuid"
+      cmake_args << "-DUUID_INCLUDE_DIRS=#{Formula["ossp-uuid"].opt_include}/ossp"
+    end
+
+    system "cmake", "-B", "build", *cmake_args
     system "cmake", "--build", "build"
     system "cmake", "--install", "build"
   end
@@ -30,7 +36,7 @@ class Drogon < Formula
       begin
         pid = fork { exec "build/hello" }
         sleep 1
-        result = shell_output("curl -s 127.0.0.1")
+        result = shell_output("curl -s 127.0.0.1:5555")
         assert_match "<hr><center>drogon", result
       ensure
         Process.kill("SIGINT", pid)

